### DISCUSSION TITLE
[Suggestion] Rainbow Glow Effect

### DISCUSSION THREAD

**Discussion by gaRt3n** [Zrips/CMI#1566] [Enhancement]
Rainbow Glow Effect

A glow rotation would be awesome

cmi.glow.rainbow

like here for example but not so fast
https://youtu.be/Os6Wr-kftrI?t=155

**Comment by Sikatsu** [comment]
I'd like to see this as well, would this be possible? @Zrips

**Comment by rapofil** [comment]
ideally option to create own effects + blinking etc.  🙏

**Comment by robomanSK** [comment]
It can be very usefull

**Comment by rapofil** [comment]
it would be EULA friendly

**Comment by RUMANGARLIC** [comment]
I think it's a good idea. It would be good to able to set their own effects

**Comment by FrandaPyco** [comment]
nice idea, EULA friendly! @Zrips any answer?
